Task3/Task3_Atanas_Kisyov/README.md

```diff
@@ -0,0 +1,13 @@
+# Connect to PostgreSQL with Python
```

Comment:
General comment. Steps un readme. I am a user and i would like to execute your program. What do I need to make? Will it work if i download a code and execute it?